tests/garbage_collect/test_garbage_collect.py

```diff
@@ -151,3 +153,191 @@ def test_entry_point(mock_logger):
         created_time,
     )
     mock_logger.info.assert_any_call("Temporary repo(s) deletion completed")
+    mock_logger.info.assert_any_call("all-rpm-content repo unavailable")
+
+def test_add_arc_args():
+    """adds the arg to the PulpTask parser"""
+    gc = GarbageCollect()
+    arg = ["", "--pulp-url", "http://some.url", "--arc-threshold", "7"]
+
+    with patch("sys.argv", arg):
+        gc_args = gc.args
+
+    assert hasattr(gc_args, "arc_threshold")
+    assert gc_args.arc_threshold == 7
+
+def test_arc_garbage_collect(mock_logger):
+    """deletes the arc repo that confirms to garbage collect criteria"""
+    repo = Repository(
+        id="all-rpm-content",
+        created=_get_created(7),
+    )
+    controller = _get_fake_controller(repo)
+    client = controller.client
+    assert list(client.search_content()) == []
+
+    all_rpm_content = client.get_repository("all-rpm-content").result()
+    existing_rpm1 = RpmUnit(
+        cdn_published=datetime.datetime.utcnow(),
+        arch="src",
+        filename="test-arc01-1.0-1.src.rpm",
+        md5sum="ba9257ced24f77f4d777e399e67924f5",
+        name="test-arc01",
+        version="1.0",
+        release="1",
+        provides=[],
+        requires=[
```

Comment:
You might consider leaving some of these fields out, particularly `requires`, just so the test doesn't need as much code.  These fields are irrelevant to this specific test.